src/webui/src/static/model/trialmanager.ts

```diff
@@ -252,7 +262,8 @@ class TrialManager {
         return updated;
     }
 
-    private async updateMetrics(lastTime?: boolean): Promise<boolean> {
+    private async updateMetrics(lastTime?: boolean): Promise<boolean | void> {
+        // private async updateMetrics(lastTime?: boolean): Promise<boolean> {
```

Comment:
Remove unused code. Why should it be `boolean | void`?

Reply:
add void type to fix this error:
![image](https://user-images.githubusercontent.com/61399850/88766707-14fd9f80-d1ab-11ea-87ef-f42d0d609c13.png)
had delete line 266.

Reply:
finally, use line 265. Thanks.